### ISSUE TITLE
Alternatives?

### ISSUE THREAD

**Issue by krazyjakee** [LeaVerou/prefixfree#209]
Unanswered pull requests and issues. No updates for nearly a year.

Any good alternatives that achieve the same but are maintained regularly?

**Comment by LeaVerou** [comment]
Is asking on a project’s repo for alternatives to that project your childish passive-aggressive way of asking for updates, or you’re just being rude?

In any case, use Autoprefixer and GTFO.

**Comment by krazyjakee** [comment]
Oops!
I didn't really think it was rude, I just considered this place to have lurkers who were in a similar position to me and may be able to help.
I gave my reasons, although perhaps too blunt.

**Comment by wladston** [comment]
I'm also interested about alternatives, or at least a way to build the minified js in a smaller form. For my use case I don't need to support all use cases, just the raw basics. I wanted to use prefixfree or a prefixfree-like solution, but without increasing my total page size...

**Comment by LeaVerou** [comment]
@wladston what do you mean? Are you referring to the 2KB -prefix-free weighs?

**Comment by krazyjakee** [comment]
@wladston There could be a feature in place to "pick and choose" which prefixes are supported and produce a custom build, but such a feature would be pretty pointless as the entire library is ~2k.

**Comment by wladston** [comment]
Yes, that's the idea. 2KB is a lot of data in terms of mobile bandwidth in Brazil :) I'll try to come up with a lighter build addition to try to bring the minified size under 2KB (the current minified script un-gzipped is currently 6202 bytes long)
